custom/ilsgateway/views.py

```diff
@@ -252,7 +252,7 @@ def end_report_run(request, domain):
         rr.complete = True
         rr.has_error = True
         rr.save()
-    except ReportRun.DoesNotExist, ReportRun.MultipleObjectsReturned:
```

Comment:
@nickpell this line had the error. Whenever an exception tries to catch two things like this, but doesn't have a variable assocaited with it, it is changed to:

```
except ReportRun.DoesNotExist as xxx_todo_changeme:
    ReportRun.MultipleObjectsReturned = xxx_todo_changeme
```